python/ray/tests/kuberay/test_autoscaling_e2e.py

```diff
@@ -325,12 +331,12 @@ def testAutoscaling(self):
         # Submit two {"Custom2": 3} bundles to upscale two workers with 5
         # Custom2 capacity each.
         logger.info("Scaling up workers with request for custom resources.")
-        kubectl_exec_python_script(
+        job_logs = ray_job_submit(
```

Comment:
I assume we want to test both modes? Here we change from exec + client -> job submission instead?

Reply:
`testAutoscaling` now covers three modes of interaction --

kubectl exec (used in the CPU request part)
Ray client (used in the GPU request part)
Job submission (used in the custom resource request part)

Reply:
Added some comments in the code.